app/src/main/java/com/example/cr554/newsapp/MainActivity.java

```diff
@@ -40,22 +41,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-
+        //find the views we'll be editing
         newsList =(RecyclerView)findViewById(R.id.newsList);
         emptyView = (TextView)findViewById(R.id.noNewsToShow);
 
+        //call fetchnews() to fetch news
```

Comment:
Don't put comments in that don't bring any new information! fetchNews() speaks for itself ^^